orderbook/src/database/orders.rs

```diff
@@ -78,6 +78,24 @@ impl Database {
             .map(|_| ())
     }
 
+    pub async fn cancel_order(&self, order: &Order) -> Result<()> {
```

Comment:
Seeing as this function only needs the uid I would rather just pass that in than the whole order.

Reply:
Good find!